scripts/hooks/deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+pwd
+main() {
+    echo "Deploying..."
+
+    if [ -z "${HOOK_tag}" ]; then
+      echo "The tag is not set. Skipping deployment."
+      exit 1
+    fi
+
+    if [ -z "${HOOK_image}" ]; then
+      echo "The image is not set. Skipping deployment."
+      exit 1
+    fi
+
+    if [ -z "${HOOK_service}" ]; then
+      echo "The service is not set. Skipping deployment."
+      exit 1
+    fi
+
+    echo "Updating service ${HOOK_service}..."
+
+    cmd="docker service update --image ${HOOK_image}:${HOOK_tag} ${HOOK_service}"
+    echo "Running command: ${cmd}"
+
+    eval $cmd
```

Comment:
Either here, or in a follow-up, we should give some thought to what we'll do if this command fails.

Reply:
We could have some logic to notify an admin of a failed webhook run? Or write out the log for this run to a file and have it accessible through a URL? But that will likely get out of scope of the current pr so I'll leave this for now.

Reply:
Docker swarm can rollback, which might be the right first move.

I think pinging an admin is out of scope for this script.  But we should make sure we "do the right thing" if the command fails and not just crash.

Reply:
[Resolved](https://github.com/Seneca-CDOT/starchart/pull/57/files#diff-68418ea51a029f95dee04a42bea2ffced96d0a3bdd351427aa2e0bece3f80320R27). The command invocation now branches into 2 separate paths.